public/js/home.js

```diff
@@ -227,214 +227,1129 @@ module.exports = function createURL(str) {
 };
 
 },{}],5:[function(require,module,exports){
-'use strict';
-
+(function (process){
 /**
  * Copyright (c) 2013-present, Facebook, Inc.
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
cc @ryanburgess this was the issue I was talking about when we last met up and I saw an awkward mismatch in the minified bundles. It looks like our current public files are using an older version of React? My minified changes are using the newer React with the MIT license. We SHOULD be on react ^15.3.1 which should have the new MIT license so I'm not sure what is causing these changes.